zigpy/types/basic.py

```diff
@@ -171,7 +177,7 @@ def serialize(self) -> bytes:
         if self._bits % 8 != 0:
             raise TypeError(f"Integer type with {self._bits} bits is not byte aligned")
 
-        return self.to_bytes(self._bits // 8, "little", signed=self._signed)
+        return self.to_bytes(self._bits // 8, self._byteorder, signed=self._signed)
```

Comment:
I haven't find any reference here or in `zha-device-handlers` but maybe a breaking change if somewhere extending `FixedIntType` and not initializing class with `byteorder="little"`.

Reply:
I think they are not supposed to extend `FixedIntType`

Reply:
Anyway, I've made little-endian the default.